src/inmanta/deploy.py

```diff
@@ -482,8 +482,15 @@ def stop(self) -> None:
         loud_logger = logging.getLogger("tornado")
         loud_logger.propagate = True
 
+        join_list: List[subprocess.Popen] = []
         if hasattr(self, "_server_proc"):
             self._server_proc.terminate()
+            join_list.append(self._server_proc)
 
         if hasattr(self, "_postgresproc"):
             self._postgresproc.stop()
+            join_list.append(self._postgresproc)
```

Comment:
Not too sure about the typing for the list PostgresProc vs subprocess.Popen

Reply:
thanks, good catch